### PR TITLE
Fix network config - Closes #3466

### LINKED ISSUE #3466: Update network config and update_config script accordingly
### Expected behavior
Network config should be extensible to coming changes and update_config script should be updated for the release 2.0

### Actual behavior
Network config is outdated, and update_config is not up to date

### Which version(s) does this affect? (Environment, OS, etc...)
2.0-

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?
Network config was outdated with the new structure.

### How did I fix it?
Update the keys for the network configs and fix the update_config script.

### How to test it?
Create sample script to migrate
`LISK_NETWORK=mainnet node scripts/update_config.js config_sample.json 1.6.0 2.0.0`
Also, check the config keys

### Review checklist

* The PR resolves #3466 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/src/modules/network/defaults/config.js [hunk 1]
```diff
@@ -8,16 +8,13 @@ const defaultConfig = {
 			env: 'LISK_WS_PORT',
 			arg: '--port,-p',
 		},
-		address: {
+		hostAddress: {
```
Comment (nazarhussain) [inline framework/src/modules/network/defaults/config.js]: Why we need to change the config, not the usage? Similar name used in the http_api module as well, should we change it there as well?

File: framework/src/modules/network/network.js [hunk 2]
```diff
@@ -61,16 +61,16 @@ module.exports = class Network {
...
+		const blacklistedPeers = this.options.blacklistedPeers
 			? this.options.access.blackList.map(peer => ({
 					ipAddress: peer.ip,
 					wsPort: peer.wsPort,
 			  }))
 			: [];
 		const p2pConfig = {
 			nodeInfo: initialNodeInfo,
-			hostAddress: this.options.address,
+			hostAddress: this.options.hostAddress,
```
Comment (nazarhussain) [inline framework/src/modules/network/network.js]:
```suggestion
			hostAddress: this.options.address,
```

We are using `address` config option for p2p layer from start, no need to change it just for underlying library.

File: lisk/scripts/update_config.js [hunk 3]
```diff
@@ -286,11 +292,29 @@ history.version('1.6.0-rc.0', version => {
...
+		config = moveElement(
+			config,
+			'peers.access.blackList',
+			'modules.network.blacklistedPeers'
+		);
+		config = moveElement(
+			config,
+			'peers.options.timeout',
+			'modules.network.connectTimeout'
+		);
```
Comment (nazarhussain) [inline lisk/scripts/update_config.js]: You also renamed `httpServerSetTimeout` to `ackTimeout`, Please add an entry for this as well.

Comment (shuse2) [reply lisk/scripts/update_config.js]: actTimeout and httpServerSetTimeout is not related, and httpServerSetTimeout was not used

File: framework/src/modules/network/network.js (hunk 2 above)
Comment (shuse2) [reply framework/src/modules/network/network.js]: This value is not very useful, but I think it's better to keep it consistent?

Comment (nazarhussain) [reply framework/src/modules/network/network.js]: Yes keep it consistent to what it was before, if we see `address` to be changed to `hostAddress` then we will do it for http_api module as well.

Comment (nazarhussain) [inline framework/src/modules/network/network.js]: Seems you forget to revert this change; 

```suggestion
			hostAddress: this.options.address,
```

File: framework/test/fixtures/config/devnet/config.json [hunk 4]
```diff
@@ -439,7 +439,7 @@
 			}
 		},
 		"network": {
-			"list": [
+			"seedPeers": [
 				{
 					"ip": "127.0.0.1",
```
Comment (jondubois) [inline framework/test/fixtures/config/devnet/config.json]: Do we want to keep the `ip` field or rename it to `ipAddress` to match the p2p library?
Either way is OK for me, just something to think about.

**Pull request merged**